fineract-provider/src/main/java/org/apache/fineract/portfolio/client/service/ClientReadPlatformServiceImpl.java

```diff
@@ -225,6 +228,7 @@ private String buildSqlStringFromClientCriteria(final SearchParameters searchPar
 
         String extraCriteria = "";
         if (sqlSearch != null) {
+        	SQLInjectionValidator.validateSQLInput(sqlSearch);
```

Comment:
This check should done for all the string params that are received for eg. in this class it should be checked for externalId, name and orderBy etc......

Also for completeness these checks needs to be done across all read platform services